x-pack/test/alerting_api_integration/spaces_only/tests/alerting/bulk_edit.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import type { SanitizedRule } from '@kbn/alerting-plugin/common';
+import { Spaces } from '../../scenarios';
+import { checkAAD, getUrlPrefix, getTestRuleData, ObjectRemover } from '../../../common/lib';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function createUpdateTests({ getService }: FtrProviderContext) {
```

Comment:
Can we add a functional test for adding an action to ensure that all the reference extracting/injecting works as expected?

Reply:
I think I already have tests that check adding actions here: 
https://github.com/elastic/kibana/blob/2e6d918e3c7694c2e129c9f272ea3af8ec75b77e/x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/bulk_edit.ts#L112-L119

It was done in `security_and_spaces`, not in `spaces_only` to follow the same fashion as similar tests for `update` methos